apps/web/src/pages/Teams/TeamMembersPage/components/useAvailableUsers.ts

```diff
@@ -1,44 +1,69 @@
 import { useQuery } from "urql";
-import React from "react";
 
-import { UserPublicProfile, graphql } from "@gc-digital-talent/graphql";
-import { notEmpty } from "@gc-digital-talent/helpers";
+import {
+  UserPublicProfile,
+  UserPublicProfileFilterInput,
+  graphql,
+} from "@gc-digital-talent/graphql";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
 
 import { TeamMember } from "~/utils/teamUtils";
 
 const TeamMembers_AvailableUsersQuery = graphql(/* GraphQL */ `
-  query TeamMembers_AvailableUsers {
-    userPublicProfiles {
-      id
-      firstName
-      lastName
-      email
```

Comment:
Query now unneeded, so can delete it from schema and unused operations

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/2642b0e1-47c6-491b-889d-b1030d0e6ee6)

Reply:
🤦‍♀️ Forget we discussed that in standup. Removed in [remove old user public profiles query](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9296/commits/9591274888f921020a5ada2f4beae6cc6847ecce)